content/manager/bootstrapping.md

```diff
@@ -147,17 +146,19 @@ After downloading the manager resources package, and placing it in an accessible
 manager_resources_package: http://my-fileserver:8080/cloudify-manager-resources_3.4.0-ga-b400.tar.gz
 {{< /gsHighlight >}}
 
-
 # Bootstrap the Manager
 
 Finally, run the `cfy bootstrap` command, pointing it to the manager blueprint file and the inputs YAML file, like so:
 
 {{< gsHighlight  sh  >}}
 $ cfy bootstrap --install-plugins -p /path/to/manager/blueprint/file -i /path/to/inputs/yaml/file
 ...
-
 {{< /gsHighlight >}}
 
+{{% gsInfo title="Info" %}}
+If you are using the CLI environment that has been installed by the official CLI RPM, and are using the `--install-plugins` parameter, you will have to execute the `bootstrap` command using `sudo`. That is because the Cloudify Python virtual environment (located in `/opt/cfy/embedded`) is owned by `root`. Alternatively, you could change the ownership of `/opt/cfy/embedded` (and its descendants) so the user running the bootstrap has write-level access.
+{{% /gsInfo %}}
+
```

Comment:
If you are using ~the~ a CLI environment that ~has been~ **was** installed ~by~ **using** the official CLI RPM, and are using the `--install-plugins` parameter, you ~will have to~ **must** execute the `bootstrap` command using `sudo` ~. That is~ because the Cloudify Python virtual environment (located in `/opt/cfy/embedded`) is owned by `root`. Alternatively, you~ could~ **can** change the ownership of `/opt/cfy/embedded` (and its descendants) so the user running the bootstrap has write-level access.